src/pages/overlay/components/activationButtons/ActivationButtons.tsx

```diff
@@ -9,15 +9,17 @@ import Ambassadors from '../../../../assets/activationButtons/ambassadors.jpg';
 interface ActivationButtonProps {
   toggleShowAlveusIntro: () => void;
   toggleShowAmbassadorList: () => void;
+  isAlveusIntroActive: boolean;
+  isAmbassadorListActive: boolean;
 }
 
 export default function ActivationButtons(props: ActivationButtonProps) {
   return (
     <div className={styles.activationButtons}>
-        <button onClick={props.toggleShowAlveusIntro}>
+        <button onClick={props.toggleShowAlveusIntro} className={`${props.isAlveusIntroActive && styles.highlighted}`}>
```

Comment:
Won't this result in `class="false"` when not active?

Reply:
Yes you're correct, fixed.
Thanks

Reply:
Do you know of any way to not define a className if there prop resolves to false? Currently, it will give the element `class=""`. I thought to render a different instance of the button without `className ` in a ternary just to avoid the empty class, but that feels a bit verbose

Reply:
You can set a prop to `undefined` if you don't want to render it normally

Reply:
Got it 👍